src/Tests/Tests/QueryDsl/TermLevel/Prefix/PrefixQueryUsageTests.cs

```diff
@@ -47,5 +52,16 @@ public class PrefixQueryUsageTests : QueryDslUsageTestsBase
 				.Value("proj")
 				.Rewrite(MultiTermQueryRewrite.TopTerms(10))
 			);
+
+		//hide
+		[U] public void DeserializeShortForm()
+		{
+			var json = JsonConvert.SerializeObject(new { description = "project description" });
```

Comment:
Can we move this to `QueryDslUsageTestsBase` and reuse in the other `DeserializeShortForm` ?

Reply:
addressed in 306aff9e3bc5418aad7f058ab1e9a1469530b9e0